release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -399,6 +399,13 @@ For more information, refer to the following sources:
 * Installer-provisioned infrastructure: xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#modifying-install-config-for-dual-stack-network_ipi-install-installation-workflow[Deploying with dual-stack networking]
 * User-provisioned infastructure: xref:../installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc#installation-configuration-parameters-network_installing-bare-metal-network-customizations[Network configuration parameters]
 
+[id="ocp-4-13-ovn-kubernetes-plugin-secondary-network"]
+==== OVN-Kubernetes is available as a secondary network (Technology Preview)
+
+With this release, the {openshift-networking} OVN-Kubernetes network plugin allows the configuration of secondary network interfaces for pods. As a secondary network, OVN-Kubernetes supports a layer 2 (switched) topology network. This is available as a Technology Preview feature.
```

Comment:
```suggestion
With this release, the {openshift-networking} OVN-Kubernetes network plug-in allows the configuration of secondary network interfaces for pods. As a secondary network, OVN-Kubernetes supports a layer 2 (switched) topology network. This is available as a Technology Preview feature.
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p  (There have been a few discussions about 'plugin' versus 'plug-in'...I assume the latest directive is to use 'plug-in'.

Reply:
I think the latest directive is to use "plugin" lol: https://redhat-documentation.github.io/supplementary-style-guide/#plugin